src/openms/source/DATASTRUCTURES/QTCluster.cpp

```diff
@@ -433,22 +438,36 @@ namespace OpenMS
           df_it != n_it->second.end(); ++df_it)
```

Comment:
if you know what is happening in the loops can you add a comment on top of them?